mods/cnc/rules/palettes.yaml

```diff
@@ -23,6 +23,10 @@
 		Filename: temperat.pal
 		ShadowIndex: 4
 		AllowModifiers: false
+	PaletteFromGimpFile:
```

Comment:
This needs to be `PaletteFromGimpOrJascFile`, right?

Reply:
Stupid me, of course.